test/DynamoCoreWpfTests/WorkspaceOpeningTests.cs

```diff
@@ -100,7 +100,7 @@ public void OpeningWorkspaceWithManualRunState()
         {
             var ws = (HomeWorkspaceModel)OpenWorkspaceInManualModeFromSampleFile(true);
             Assert.AreEqual(ws.RunSettings.RunType, RunType.Manual);
-            Assert.IsFalse(ws.HasRunWithoutCrash);
+            Assert.IsTrue(ws.HasRunWithoutCrash);
         }
```

Comment:
Workspace is no longer exhibiting a crash during this test?

Reply:
@alfarok I think you misinterpreting that - I believe the idea was that this sample file used to have ws.HasRunWithoutCrash set to false, but now it's changed? Can you check that?

Reply:
@mjkkirschner `ws.HasRunWithoutCrash = false` means the graph was crashing no?

Reply:
@mjkkirschner Ah so if this graph was set to run in manual it would not have executed therefore `HasRunWithoutCrash` is set to false because the graph hasn't run but hasn't actually crashed.

Reply:
This test is failing because of [this issue](https://jira.autodesk.com/browse/QNTM-2839) we original saw when we ran all the tests against JSON